src/content_sao.cpp

```diff
@@ -811,13 +812,15 @@ PlayerSAO::PlayerSAO(ServerEnvironment *env_, RemotePlayer *player_, session_t p
 	m_prop.colors.emplace_back(255, 255, 255, 255);
 	m_prop.spritediv = v2s16(1,1);
 	m_prop.eye_height = 1.625f;
-	// end of default appearance
+	// End of default appearance
 	m_prop.is_visible = true;
 	m_prop.makes_footstep_sound = true;
 	m_prop.stepheight = PLAYER_DEFAULT_STEPHEIGHT * BS;
-	m_prop.can_zoom = true;
 	m_hp = m_prop.hp_max;
 	m_breath = m_prop.breath_max;
+	// Allow the client to set a limited range of zoom FOV, a more powerful
+	// zoom can only be set by server mods.
+	m_prop.zoom_fov = rangelim(g_settings->getFloat("zoom_fov"), 15.0f, 160.0f);
```

Comment:
This means that the server will force its fov setting, as soon the ObjectProperties are sent do the player.